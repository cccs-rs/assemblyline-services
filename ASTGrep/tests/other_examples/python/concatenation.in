var1 = "aaa"
var2 = "bbb"
var3 = "ccc"

func1(var1 + var2 + var3)

var4 = b"ddd"
var5 = b"eee"
var6 = b"fff"

func2(var4 + var5 + var6)
