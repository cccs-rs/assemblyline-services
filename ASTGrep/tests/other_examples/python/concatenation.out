var1 = "aaa"
var2 = "bbb"
var3 = "ccc"

func1('aaabbbccc')

var4 = b"ddd"
var5 = b"eee"
var6 = b"fff"

func2(b'dddeeefff')
